frontend/packages/kubevirt-plugin/src/selectors/immutable/template/combined.ts

```diff
@@ -99,3 +104,24 @@ export const iGetCommonTemplateCloudInit = (tmp: ITemplate) => {
   );
   return iGet(iCloudInitStorage, 'cloudInitNoCloud');
 };
+
+export const iGetDefaultTemplate = (
+  iCommonTemplates: ImmutableMap<string, ITemplate>,
+  os: string,
+): ITemplate =>
+  (iCommonTemplates || ImmutableMap()).valueSeq().find((iTemplate) => {
+    const labels = iGetLabels(iTemplate);
+    return labels && labels.get(getOsLabel(os)) && labels.get(TEMPLATE_DEFAULT_LABEL) === 'true';
+  });
+
+export const getITemplateDefaultFlavor = (template: ITemplate): Flavor =>
+  template &&
+  Flavor.getAll().find(
```

Comment:
there is no need to go all over them - there is just one flavor per template

Reply:
and how would you detect the right one?

the label is `flavor.template.kubevirt.io/FLAVOR`
we need to iterate through all of them and find the one who matches.

Reply:
it is a map with well defined key so it should be easy to detect

Reply:
sorry, you are right, this is the correct solution. I mistakenly had in my mind that we know the flavor/workload